landscape.yml

```
@@ -378,6 +378,11 @@ landscape:
             repo_url: https://github.com/eiffel-community/eiffel
             logo: eiffel.svg
             crunchbase: https://www.crunchbase.com/organization/eiffel-community
+          - item:
+            name: CD Events
+            description: CD Events is an event protocol providing interoperability in CD systems
```

We need to align to other places where spell out the name. Is this the correct form?
```suggestion
            name: CDEvents
            description: CDEvents is an event protocol providing interoperability in CD systems
```

---

Correct, I will add that in my coming update